src/coffee/stockimport.coffee

```diff
@@ -163,7 +163,8 @@ class StockImport
               expectedDelivery = new Date(committedDeliveryDate).toISOString()
             catch error
               @logger.warn "Can't parse date '#{committedDeliveryDate}'. Creating entry without date..."
-          d = @_createInventoryEntry(sku, appointedQuantity, expectedDelivery, channelId)
+          restockableInDays = undefined
```

Comment:
Is this variable needed? we can pass `undefined` directly to `_createInventoryEntry()` function.

Reply:
Yes, it is needed.
If it is not declared above as `undefined`, `restockableInDays` will cause a reference error when @_createInventoryEntry() is called.